components/daterangepicker/overview.md

```diff
@@ -54,26 +54,27 @@ To restrict the user from writing dates in the input so that the end is after th
-| `Enabled` | `bool` | Specifies whether typing in the input is allowed.
-| `Format` | `string` | Specifies the format of the DateInputs of the DateRangePicker. [Read more about supported data formats in Telerik DateInput for Blazor UI]({%slug components/dateinput/supported-formats%}) article.
-|`EndId` and `StartId` | `string` | render as the `id` attribute on the `<input />` element, so you can attach a `<label for="">` to the input.
-| `Min` | `DateTime` | The earliest date that the user can select.
-|`Max` | `DateTime` | The latest date that the user can select.
-| `StartValue` and `EndValue` | `T` | The current values of the inputs for start and end of the range. Can be used for two-way binding.
-| `View` | ` CalendarView` | Specifies the current view that will be displayed in the popup calendar.
-| `TabIndex` | `int?` | maps to the `tabindex` attribute of both `input` HTML elements in the component and them both will have the same `tabindex`. You can use it to customize the order in which the inputs in your form focus with the `Tab` key.
-| `Placeholder` |`string` | maps to the `placeholder` attribute of the HTML element. The `Placeholder` will appear if the component is bound to **nullable** DateTime object - `DateTime?`, but will not be rendered if the component is bound to the default value of a non-nullable DateTime object. The Placeholder value will be displayed when the input is not focused. Once the user focuses it to start typing, the Format Placeholder (default or [customized one](#format-placeholder)) will override the Placeholder to indicate the format the date should be entered in.
+| `BottomView` | ` CalendarView` <br/> (`CalendarView.Month`) | Defines the bottommost view in the popup calendar to which the user can navigate to. |
+| `DebounceDelay` | `int` <br/> (`150`) | Time in milliseconds between the last typed symbol and the value update. Use it to balance between client-side performance and number of database queries. |
+| `DisabledDates` | `List<DateTime>` | A list of dates that can not be selected as the start or end of the range, see the <a href="https://demos.telerik.com/blazor-ui/daterangepicker/disabled-dates" target="_blank">Live Demo: Date Range Picker Disabled Dates</a>. |
+| `Enabled` | `bool` | Whether typing in the input is allowed. |
+| `Format` | `string` | The format of the DateInputs of the DateRangePicker. [Read more about supported data formats in Telerik DateInput for Blazor UI]({%slug components/dateinput/supported-formats%}) article. |
+|`EndId` and `StartId` | `string` | The `id` attribute on the `<input />` element, so you can attach a `<label for="">` to the input. |
+| `Min` | `DateTime` | The earliest date that the user can select. |
+| `Max` | `DateTime` | The latest date that the user can select. |
+| `Orientation` | `CalendarOrientation` <br/> (`CalendarOrientation.Horizontal`) | The orientation of the calendar popup. The available options are `Horizontal` and `Vertical`. |
```

Comment:
```suggestion
| `Orientation` | `CalendarOrientation`enum <br /> (`Horizontal`) | The orientation of the calendar popup when using multiple views. The available options are `Horizontal` and `Vertical`. |
```

Reply:
Actually, I had to revert this. "When using multiple views" is not applicable for the DateRangePicker - it does not support multi-view, it always shows two calendars by default.